src/representative/rep_periods.jl

```diff
@@ -0,0 +1,200 @@
+
+"""
+    AbstractRepresentativePeriod{T} <: TimeStructure{T}
+
+Abstract type used for time structures that represent a representative period.
+These periods is obtained when iterating through the representative periods of a time
+structure declared using the function [`repr_period`](@ref.)
+"""
```

Comment:
These periods are obtained when iterating through the representative periods of a time
structure using the function...